iothub_client/src/iothub_client_ll_uploadtoblob.c

```diff
@@ -81,6 +81,7 @@ typedef struct IOTHUB_CLIENT_LL_UPLOADTOBLOB_HANDLE_DATA_TAG
     
     char* certificates;
     HTTP_PROXY_OPTIONS http_proxy_options;
+    const char* curl_interface;
```

Comment:
please move new members to end of struct

Reply:
done